### ISSUE TITLE
Cannot import content script

### ISSUE THREAD

**Issue by avi12** [AWolf81/snowpack-chrome-preact-popup-demo#2]
I made a content script `src/scripts/content-script.ts`:
```js
import { hello } from "./content-script-2";
console.log(hello());
console.log("Hello from content script");
```
`src/scripts/content-script-2.ts`
```js
export function hello() {
  return "Hello World";
}
```
When I open the web page listed in the [matches](https://developer.chrome.com/docs/extensions/mv2/content_scripts/#:~:text=%22matches%22), I get:
```
Uncaught SyntaxError: Cannot use import statement outside a module
```
For some reason, I cannot import it even after listing `dist/scripts/content-script-2.js` in [`web_accessible_resources`](https://developer.chrome.com/docs/extensions/mv2/manifest/web_accessible_resources)

**Comment by AWolf81** [comment]
Yes, good point. It's not easy to do as the content-scripts in the manifest are not loaded as js modules.

A workaround would be like in the following [gist](https://gist.github.com/AWolf81/045384dd4123afce1724735da1217abd).

With this Gist, you can structure the app however you like it and use import/exports.

I think I could add something to the snowpack-plugin so it is easier to use/setup but I'm not sure how. 
Maybe I'll create the injection script and add it to the project as a utility script. Then the user can remove it from the `manifest.json` to disable it if there is no content script needed.

**Comment by avi12** [comment]
You can take a note of [Rollup's Chrome extension plugin](https://github.com/extend-chrome/rollup-plugin-chrome-extension) which supports such functionality, at least in Chrome
For Firefox, sadly, a different workaround is needed, perhaps using IIFE directly

**Comment by windusayles** [comment]
I've been successfully serving my content.js script for the past week+, and just today I noticed that I have a new error, the same as mentioned in the beginning of this Issue by @avi12.
